installer/Controller/MigrationController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+
+namespace Installer\Controller;
+
+
+class MigrationController implements Controller
+{
+
+	/**
+	 * @return array
+	 */
+	public function do()
+	{
+		$output = [];
```

Comment:
So even though hovering over the button for this step says _Creating the Database_, this doesn't actually create it. Is that impossible because we now support multiple database engines? If that's the case, then I guess an additional test step should be added after the environment is set up, to check whether the database exists?

Reply:
good point.

Reply:
but how...